Assets/Scripts/Entity/Enemy/EnemyController.cs

```diff
@@ -158,7 +166,15 @@ private void MoveToNextPoint()
             }
         }
 
-        public Area WanderArea => wanderArea;
+        /// <summary>
+        /// Run away. (Move to next point with running speed.)
+        /// </summary>
+        public void RunAway()
+        {
+            _runningAway = true;
+            NavMeshAgent.speed = runningSpeed;
+            MoveToNextPoint();
```

Comment:
Probably just a random point in any direction? Or away from the attacker?

Reply:
It's just another random point